roles/openshift_health_checker/openshift_checks/docker_image_availability.py

```diff
@@ -48,14 +48,20 @@ def run(self, tmp, task_vars):
         if failed:
             if "Error connecting: Error while fetching server API version" in msg:
                 msg = (
-                    "It appears Docker is not running.\n"
+                    "It appears Docker is not running. This check will be skipped.\n"
                     "Please start Docker on this host before running this check.\n"
                     "The full error reported was:\n  " + msg
                     )
+
+                return {
+                    "failed": False,
+                    "changed": changed,
+                    "msg": msg,
+                }
```

Comment:
This is silencing the error instead of skipping. Normally in Ansible when `failed` is not present or `False` we don't ever look for a `msg`.

Instead, if possible, I'd use the mechanism we have for skipping checks -- implementing `is_active`. When it is not active, we automatically set `skipped: True` in the return value, and then we could present skipped checks (with perhaps a reason, which is something we may still need to add) in the UI.